app/models/pool.rb

```diff
@@ -31,19 +32,18 @@ def remove_pool
     return if name == Pool::DEFAULT_POOL
```

Comment:
Look at my comment in https://github.com/ytolstyk/hema/issues/73

Since you have this check here, remove the controller check, please.